doc/tutorials/using_ubuntu_as_sos.rst

```diff
@@ -269,4 +296,50 @@ You are now all set to start the User OS (UOS)
 
 **Congratulations**, you are now watching the User OS booting up!
 
-.. _Clear Linux: https://clearlinux.org
+
+
+Enabling network sharing 
+************************
+
+after booting up SOS and UOS, you may need to run some basic test and install addational pakcages
+enabling network in UOS is must, enabling the requiring tap and networking bridge in the SOS 
+to enable networking in the guest OS is important, you can create a script file to setup.
+
+below is an example for you reference:(verifed in Ubuntu 14.04 and 16.04 as the SOS) 
+
+.. code-block:: none
+
+  #!/bin/bash
+
+  #setup bridge for uos network
+  br=$(brctl show | grep acrn-br0)
+  br=${br-:0:6}
+
+  ip tuntap add dev acrn_tap0 mode tap
+  
+  
+  taps=$(ifconfig | grep acrn_ | awk '{print $1}')
+  
+  # if bridge not existed
+  if [ "$br"x != "acrn-br0"x ]; then
+  #setup bridge for uos network
+  brctl addbr acrn-br0
+  brctl addif acrn-br0 enp3s0
+  ifconfig enp3s0 0
+  dhclient acrn-br0
+
+  # add existing tap devices under the bridge
+    for tap in $taps; do
+      ip tuntap add dev acrn_$tap mode tap
+      brctl addif acrn-br0 $tap
+      ip link set dev $tap down
+      ip link set dev $tap up
+    done
+  fi
+  
+  brctl addif acrn-br0 acrn_tap0
+  ip link set dev acrn_tap0 up
+
+Enabling USB keyboard and mouse for UOS, please ref: https://projectacrn.github.io/latest/getting-started/apl-nuc.html  
```

Comment:
again, please use ``:ref:`` syntax to refer to the technical docs.

Reply:
please merge this first, then let me fix it,  i am not able to modify this patch directly ,  if i modify this here, it will lead to another submit,